admin_style.css

```diff
@@ -0,0 +1,45 @@
+* {
+    box-sizing: border-box
+}
+
+body {
+    font-family: Helvetica, Arial, sans-serif;
+}
+
+.container{
+    width: 90%;
+    margin: 0 auto;
+}
+
+.mainText{
+    text-align: center;
+    font-size: 24px;
+    height: 100px;
+}
+
+.forms textarea{
+    float: right;
+}
+
+.forms input{
+    float: left;
```

Comment:
This is not needed. The items are there already. If you don't want the space between them than you can leave it.

Reply:
thanks, it actually fixed my spacing problem